core/ipfs/node/node.go

```diff
@@ -0,0 +1,65 @@
+package ipfs
+
+import (
+	"os/exec"
+	"context"
+
+	"github.com/FleekHQ/space-daemon/config"
+	"github.com/FleekHQ/space-daemon/log"
+
+	process "github.com/jbenet/goprocess"
+)
+
+type IpfsNode struct {
+	proc  	  process.Process
+	IsRunning bool
+	Ready     chan bool
+	cfg       config.Config
+}
+
+func NewIpsNode(cfg config.Config) *IpfsNode {
+	return &IpfsNode{
+		Ready: make(chan bool),
+		cfg:   cfg,
+	}
+}
+
+func (node *IpfsNode) Start(ctx context.Context) error {
+    log.Printf("Starting the ipfs node")
```

Comment:
You should import `"github.com/FleekHQ/space-daemon/log"` and then use `log.Debug` instead of printing directly.

Reply:
@dmerrill6 

Great catch!

I just copied `log.Printf` without any consideration from [core/sync/sync.go](https://github.com/FleekHQ/space-daemon/blob/f482e013697206a55fe57cd7450a4f620495cb29/core/sync/sync.go#L100), [changed](https://github.com/FleekHQ/space-daemon/pull/112/commits/14937b1f1d98abd2ba55936c5a1080853602445c) to `log.Info`.